.env

```diff
@@ -0,0 +1,4 @@
+# mongo "mongodb+srv://cluster0-wojc5.mongodb.net/test"  --username user-one
+# mongodb://username:password@host:port/database?options...
+MONGODB_URI = mongodb+srv://user-one:password%21@cluster0-wojc5.mongodb.net/test?retryWrites=true&w=majority
```

Comment:
won't it be necessary to add this file(`.env`) in `.gitignore this file

Reply:
Thanks for reminding me!